user_accounts/migrations/0027_remove_oakland_clinic_from_addresses.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-10-12 21:18
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def delete_oakland_clean_slate_clinic(apps, schema_editor):
+    Address = apps.get_model('user_accounts', 'Address')
```

Comment:
If you only want to delete the record for Alameda Public Defender, filter for that org